### PR TITLE
Feature/search bar

### PR DESCRIPTION
**Pull request by b1u3too**
Hi Pals!

### Before: 
- `npm i`

### Expect:
- There is now a searchBar at the top of the page! It autocompletes city names. It is currently struggling with Halifax but otherwise seems to behave. Going to log the halifax bug and get to the bottom of it, leaning into mentor assistance. Endpoint seems fine, but it's misbehaving in this implementation!

### CODE REVIEW

File: leafsmart-front-end/src/App.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import Events from "./components/Events"
 import "./App.scss";
 import CityFacts from "./components/CityFacts";
 import QoLData from "./components/QoLData";
+import Search from "./components/Search";
 import './styles/main.css';
 
 const App = () => {
```
Comment (b1u3too) [inline leafsmart-front-end/src/App.js:10]: Are we ready to let go of the commented out code?

Comment (MGS31) [reply leafsmart-front-end/src/App.js:10]: I am ok with removing it as we would be using techincally something different for log in log out right?

**Review by MGS31 — APPROVED** [review]
Looks solid to me! love the use of the rapid API and I hope it helped speed things up a bit.

File: leafsmart-front-end/src/App.js (hunk 1 above)
Comment (b1u3too) [reply leafsmart-front-end/src/App.js:10]: Yep! I think log in/out would use a similar kind of idea to "talk" to our api, which all just mirrors the API calls we're making to third(and fourth... and fifth) party endpoints.

Comment (rachelpr) [reply leafsmart-front-end/src/App.js:10]: fine by me!

**Review by rachelpr — APPROVED** [review]

**Pull request merged**